.github/workflows/check-change.yml

```
@@ -18,13 +46,15 @@ jobs:
         with:
           script: |
             const beachballVersion = require('./package.json').devDependencies.beachball;
+
             if (typeof beachballVersion !== 'string') {
               core.setFailed("Please check 'package.json', it should include 'beachball' in 'devDependencies'")
-            } else {
-              console.log(`Using beachball@${beachballVersion}...`)
-              await exec.exec('yarn', ['global', 'add', `beachball@${beachballVersion}`]);
+              return
             }
 
+            console.log(`Using beachball@${beachballVersion}...`)
```

For consistency and potential futureproofing? Under the hood it's the same, wdyt?

```suggestion
            core.info(`Using beachball@${beachballVersion}...`)
```